app/javascript/react/assets/styles/Typography/Typography.style.tsx

```diff
@@ -1,34 +1,46 @@
-import styled, { css } from "styled-components";
+import styled from "styled-components";
+
+const LargeH1 = styled.h1`
```

Comment:
I see this component more as a span, that is just decorative, semantically it is not the most important information on the page, I think.
BTW there is always only one H1 element on the given page, I know right now we have none, though.

Reply:
I
<img width="290" alt="Zrzut ekranu 2023-11-23 o 20 19 09" src="https://github.com/HabitatMap/AirCasting/assets/44276070/9fa01369-5826-4
![Uploading Zrzut ekranu 2023-11-23 o 20.19.34.png…]()
724-b798-f2429ef304d1">
 changed this to span, but then the spacing between elements is odd. Can you tell me why?